src/main/java/com/iexec/sms/admin/AdminService.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.iexec.sms.admin;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class AdminService {
+
+    public String createDatabaseBackupFile() {
+        return "createDatabaseBackupFile is not implemented";
+    }
+
+    public String restoreDatabaseFromBackupFile(String storageId, String fileName) {
+        return "restoreDatabaseFromBackupFile is not implemented";
+    }
```

Comment:
Should we add basic javadoc to avoid merge conflicts ?
```suggestion
    /**
    * Creates a basic backup file...
    */
    public String createDatabaseBackupFile() {
        return "createDatabaseBackupFile is not implemented";
    }

    public String restoreDatabaseFromBackupFile(String storageId, String fileName) {
        return "restoreDatabaseFromBackupFile is not implemented";
    }
```

Reply:
I agree with you, but I think that the signature on the service side is likely to change. I think it's better for the person who codes this part to push the doc.

Reply:
ok, I agree with you. The conflict, if any will be minimal.